provisioning/README.md

```diff
@@ -265,6 +265,24 @@ WHERE {
 }
 ```
 
+### Notice for Docker Image
+
+DockerHub: https://hub.docker.com/r/tractusx/provisioning-agent
+
+Eclipse Tractus-X product(s) installed within the image:
+GitHub: https://github.com/eclipse-tractusx/knowledge-agents/tree/main/provisioning
+Project home: https://projects.eclipse.org/projects/automotive.tractusx
+Dockerfile: https://github.com/eclipse-tractusx/knowledge-agents/blob/main/provisioning/src/main/docker/Dockerfile
+Project license: Apache License, Version 2.0
+
+Used base image
+
+[ontop/ontop:5.0.2](https://hub.docker.com/r/ontop/ontop/tags?page=1&name=5.0.2)
```

Comment:
We should not only link the used image, but provide as much additional info on it as we can.
Examples on images that we already aligned on can be found in [TRG 4.06](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06)

if the info is provided in later PRs, i would appreciate a GitHub issue before merging the PR, so we don't forget about that

Reply:
I'll try to provide that today.